geode-core/src/main/java/org/apache/geode/internal/cache/GemFireCacheImpl.java

```diff
@@ -841,9 +937,9 @@ public static GemFireCacheImpl getForPdx(String reason) {
       Function<InternalCache, InternalResourceManager> internalResourceManagerFactory,
       Supplier<Integer> serialNumberSupplier,
       HeapEvictorFactory heapEvictorFactory,
-      Consumer<Void> typeRegistryInit,
-      Consumer<Void> typeRegistryOpen,
-      Consumer<Void> typeRegistryClose,
+      Runnable typeRegistryInit,
+      Runnable typeRegistryOpen,
+      Runnable typeRegistryClose,
```

Comment:
Yeah, this is much better. Later we may be able to bundle these three runnables into a single interface. And similar for any other clumps of related parameters.

Reply:
Yep, that's a great idea!